handyman/data/product_product.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="appointment_account_payment.default_booking_product" model="product.product">
+    <field name="product_tmpl_id" ref="appointment_account_payment.default_booking_product_product_template"/>
+    <field name="purchase_ok" eval="False"/>
+  </record>
+  <record id="product_product_14" model="product.product">
+    <field name="product_tmpl_id" ref="product_template_14"/>
+  </record>
+  <record id="product_product_15" model="product.product">
+    <field name="product_tmpl_id" ref="product_template_15"/>
+  </record>
```

Comment:
For those two unused product, you could simply remove the context the related product_template creation
```suggestion
```